src/commands/deploy/deploy.ts

```diff
@@ -79,14 +51,14 @@ async function deploy(context: IActionContext, target: vscode.Uri | string | Slo
         throw new Error(localize('pythonNotAvailableOnWindows', 'Python projects are not supported on Windows Function Apps.  Deploy to a Linux Function App instead.'));
     }
 
-    await validateRemoteBuild(context, client, workspaceFolder.uri.fsPath, language);
+    await validateRemoteBuild(context, node.root.client, workspaceFolder.uri.fsPath, language);
```

Comment:
Why get rid of the client variable?  You call `node.root.client` enough times that it makes sense to keep the variable.

Reply:
Idk it just felt right - maybe because I was working on line length and that was an easy way to delete a line. No strong preference